lib/Doctrine/ODM/PHPCR/Query/Query.php

```diff
@@ -180,7 +188,7 @@ public function execute($parameters = null, $hydrationMode = null)
                 $data = $this->query->execute();
                 break;
             case self::HYDRATE_DOCUMENT:
-                $data = $this->dm->getDocumentsByPhpcrQuery($this->query, $this->documentClass, $this->primaryAlias);
+                $data = $this->dm->getDocumentsByPhpcrQuery($this->query, $this->documentClass);
```

Comment:
Is it really unused? It is set in the constructor and is indeed the third argument of `getDocumentsByPhpcrQuery`.